src/main/jhipster/src/main/webapp/app/entities/sample/sample.service.ts

```diff
@@ -62,6 +62,11 @@ export class SampleService {
             .pipe(map((res: HttpResponse<Sample[]>) => this.convertArrayResponse(res)));
     }
 
+    removeEntries(sampleListId: number, selectedEntries: any) {
+        const url = SERVER_API_URL + `crops/${this.context.cropName}/sample-lists/${sampleListId}/entries?selectedEntries=${selectedEntries}&programUUID=` + this.context.programUUID;
```

Comment:
Use params object similar to this: 

https://github.com/IntegratedBreedingPlatform/Workbench/blob/0114a74a089b924aa41128b8d7af82dde4e44e8a/src/main/jhipster/src/main/webapp/app/shared/germplasm/service/germplasm.service.ts#L127